frontend/pages/_app.tsx

```diff
@@ -13,13 +15,20 @@ type AppPropsWithLayout<P> = AppProps<P> & {
 
 function MyApp({ Component, pageProps }: AppPropsWithLayout<any>) {
    const getLayout = Component.getLayout ?? ((page) => page);
+   const polyfillDomain = POLYFILL_DOMAIN ?? 'https://polyfill.io';
    return (
-      <ServerSidePropsProvider props={pageProps}>
-         <AppProviders {...pageProps.appProps}>
-            <NextNProgress showOnShallow={false} />
-            {getLayout(<Component {...pageProps} />, pageProps)}
-         </AppProviders>
-      </ServerSidePropsProvider>
+      <>
+         <Script
+            src={`${polyfillDomain}/v3/polyfill.min.js?features=Intl.RelativeTimeFormat,Intl.RelativeTimeFormat.~locale.en`}
```

Comment:
```suggestion
            src={`${polyfillDomain}/v3/polyfill.min.js?features=default,Intl.RelativeTimeFormat,Intl.RelativeTimeFormat.~locale.en`}
```
Is there any reason we want to only polyfill this one thing, rather than let the polyfill service do it's thing and polyfill any of the (common) things that are missing?

Reply:
Seems like a reasonable thing to do!